oasis-std/src/lib.rs

```diff
@@ -59,3 +101,23 @@ impl AddressExt for Address {
         crate::backend::code(self).unwrap()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    extern crate oasis_test;
+
+    #[test]
+    fn test_invoke() {
+        invoke!(
```

Comment:
Should this check the backend is updated correctly?

Reply:
memchain is 💩 and unable to do anything useful